app/workers/delete_object_hierarchy_worker.rb

```diff
@@ -53,7 +53,7 @@ def on_finish(method_name, options)
   def build_batch
     batch = Sidekiq::Batch.new
     batch.description = batch_description
-    batch_callbacks(batch) { batch.jobs { delete_associations(object) } }
+    batch_callbacks(batch) { batch.jobs { delete_associations } }
```

Comment:
because `object` is [an instance variable](https://github.com/3scale/porta/blob/de3b4578961b7b1c6ccf8cf3f315a84943aec7ef/app/workers/delete_object_hierarchy_worker.rb#L51).